.github/workflows/test-command.yml

```diff
@@ -300,52 +302,43 @@ jobs:
     # Initialize the terratest go project
     - name: "Initialize terratest Go project"
       run: make -C test/src clean init
-
-    # Run the terratest integration tests without credentials
-    - name: "Test `examples/complete` with terratest locally, without credentials"
-      if: ${{ contains(github.event.client_payload.github.payload.repository.name, 'terraform-null-') }}
-      run: make -C test/src
-
-    # Run the terratest integration tests with only AWS credentials
-    - name: "Test `examples/complete` with terratest on AWS"
-      if: ${{ contains(github.event.client_payload.github.payload.repository.name, 'terraform-aws-') && !( contains(github.event.client_payload.github.payload.repository.name, '-github-') || contains(github.event.client_payload.github.payload.repository.name, '-datadog-') || contains(github.event.client_payload.pull_request.labels.*.name, 'terraform-github-provider') || contains(github.event.client_payload.pull_request.labels.*.name, 'terraform-datadog-provider') ) }}
-      run: make -C test/src
+    - name: "Inject secrets"
       env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
+        USES_GITHUB:   ${{ contains(github.event.client_payload.github.payload.repository.name, '-github-') }}
```

Comment:
Not all of our modules that rely on github unfortunately have it in the repo name. We need a condition similar to the datadog one that also checks for labels.